cfgov/unprocessed/apps/owning-a-home/js/explore-rates/util.js

```diff
@@ -109,7 +109,12 @@ function renderAccessibleData( tableHead, tableBody, labels, vals ) {
  * @param {string} time - Timestamp from API.
  */
 function renderDatestamp( elem, time ) {
-  elem.textContent = formatTimestampMMddyyyy( time );
+  let formattedTime;
+  if ( time ) {
+    formattedTime = formatTimestampMMddyyyy( time );
+  }
+
+  elem.textContent = formattedTime;
```

Comment:
This will set `elem.textContent` to undefined if `time` is undefined, won't it? What's the scenario where time would be undefined and do we want another default behavior here in that case?